bindings/pyroot/pythonizations/python/ROOT/_pythonization/_th1.py

```diff
@@ -23,10 +24,55 @@ def _imul(self, c):
     return self
 
 
-@pythonization('TH1')
+_np_dtype_dict = {
+    "float64": "TH1D",
+    "float32": "TH1F",
+    "int32": "TH1I",
+    "int8": "TH1C",
+    "int16": "TH1S",
+}
+
+
+def FromNumpy(*args, **kwargs):
+    r"""Function to create histogram object from Numpy arrays"""
+    th, npval = _numpy_getter(_np_dtype_dict, *args, **kwargs)
+    th.FillN(npval[0].size, *npval)
+    return th
```

Comment:
This new `FromNumpy` function would need way more info in the docstring, because it's not clear how it should used otherwise. What should be passed as `args` and `kwargs`?

I understand what it does only after reading the source code. And I think it mixes the constructor with the the filling a bit too much. I think the interface would be clearer (e.g. without having to accept the numpy array as an arbitrary additional constructor argument if you instead pythonize `TH1::Fill()` to take numpy arrays. And that would be good anyway.

What do you think @lmoneta?

Reply:
@guitargeek Yes sir I will be adding more documentation soon for arguments to be passed, thanks

Reply:
> I understand what it does only after reading the source code. And I think it mixes the constructor with the the filling a bit too much. I think the interface would be clearer (e.g. without having to accept the numpy array as an arbitrary additional constructor argument if you instead pythonize `TH1::Fill()` to take numpy arrays. And that would be good anyway.
 
 I agree, to this, although in the beginning the function was intended for converting args and passing them as numpy arrays in FillN()